src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -385,11 +393,29 @@ else if ( Double.isInfinite(expected) ) // NaN == NaN => false unfortunately
     }
 
     private static void assertAttributeEquals(final String key, final Object actual, final Object expected) {
-        if ( expected instanceof Double ) {
+        Object notationCorrectedActual = normalizeScientificNotation(actual);
+        Object notationCorrectedExpected = normalizeScientificNotation(expected);
+        if (notationCorrectedExpected instanceof Double && notationCorrectedActual instanceof Double) {
             // must be very tolerant because doubles are being rounded to 2 sig figs
-            assertEqualsDoubleSmart(actual, (Double) expected, 1e-2);
-        } else
-            Assert.assertEquals(actual, expected, "Attribute " + key);
+            assertEqualsDoubleSmart((Double) notationCorrectedActual, (Double) notationCorrectedExpected, 1e-2, "Attribute " + key);
+        } else {
+            Assert.assertEquals(notationCorrectedActual, notationCorrectedExpected, "Attribute " + key);
+        }
+    }
+
+    /**
+     * This is some  necessary to deal with the fact that variant context attributes are deserialized from vcf as Strings
```

Comment:
Should we move these variantcontext methods as well?  It seems like they should all go somewhere else.